Frontend/src/app/components/data-lab/active-indicator-card/active-indicator-card.component.html

```
@@ -0,0 +1,52 @@
+<div class="aic" (click)="onConfigure($event)" role="button" tabindex="0"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)